src/ocm/components/clusterConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -76,25 +76,22 @@ const NetworkConfigurationForm: React.FC<{
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [infraEnvError]);
 
-  const handleSubmit: FormikConfig<NetworkConfigurationValues>['onSubmit'] = async (
-    values,
-    actions,
-  ) => {
+  const handleSubmit: FormikConfig<NetworkConfigurationValues>['onSubmit'] = async (values) => {
     clearAlerts();
 
     // update the cluster configuration
     try {
       const isMultiNodeCluster = !isSNO(cluster);
       const isUserManagedNetworking = values.managedNetworkingType === 'userManaged';
-      const params = omit(values, [
-        'hostSubnet',
-        'useRedHatDnsService',
-        'managedNetworkingType',
-        'machineNetworkCidr',
-        'serviceNetworkCidr',
-        'clusterNetworkCidr',
-        'clusterNetworkHostPrefix',
-      ]);
+
+      const params: V2ClusterUpdateParams = {
+        apiVip: values.apiVip,
+        ingressVip: values.ingressVip,
+        sshPublicKey: values.sshPublicKey,
+        vipDhcpAllocation: values.vipDhcpAllocation,
+        networkType: values.networkType,
+      };
+
```

Comment:
more defensive approach - just pick the values we know we want.

Reply:
We don't need edit_proxy and enable_proxy? we currently send them in the patch request

Reply:
no we dont. We were sending them because of that `omit` usage. But they are not a values we sent to the backend (and its espacially true for OCM which does not use these at all, only ACM)